nengo_dl/simulator.py

```diff
@@ -577,18 +583,38 @@ def reset(self, seed=None):
         seed : int
             If not None, overwrite the default simulator seed with this value
             (note: this becomes the new default simulator seed).
+        include_trainable : bool
+            If True (default), also reset any online or offline training that has been
+            performed on simulator parameters (e.g., connection weights).
```

Comment:
I had to manually copy over the changes to `include_trainable`'s docstring during the rebase. Just want to double-check that this is correct (i.e. by default, `sim.reset` will reset both online and offline training).

Reply:
Yep that's right!